app/entities/user.go

```diff
@@ -87,3 +90,13 @@ func checkPass(password string) error {
 
 	return nil
 }
+
+var idRegex = regexp.MustCompile(IDMask)
+
+func ValidateUUID(id string) error {
+	if valid := idRegex.MatchString(id); !valid {
+		return globals.ErrInvalidID
+	}
+
+	return nil
+}
```

Comment:
This is in file user.go but does UUID only applicable only for user?

Reply:
Yep. Good reason. Fixed